src/__tests__/index.js

```diff
@@ -386,22 +384,15 @@ test('passes an updated theme when theme prop changes', () => {
   expect(wrapper).toMatchSnapshotWithGlamor(`with theme prop of padding 20px`)
 })
 
-test('cleans up theme subscription when unmounts', () => {
-  const unsubscribe = jest.fn()
-  const context = getMockedContext(unsubscribe)
+test('passes `theme` to the css prop if it is a function', () => {
```

Comment:
Added (sorry the git diff makes this a little confusing).

Incidentally, I added this test because I thought I could get away with only wrapping `GlamorousComponent` with `withTheme` if any of the `...styles` was a function. So get a little bit of an optimization by not wrapping it for plain old objects. Turns out I can't do that because the `css` prop can be a function which accepts the `theme` and you can't know that ahead of time. So all components need to keep track of the theme. #sigh